application-migrator-migrators/application-migrator-migrators-document/application-migrator-migrators-document-default/src/main/java/org/xwiki/contrib/migrator/migrators/internal/WikiDocumentMigrationDescriptorProvider.java

```diff
@@ -48,21 +48,19 @@
     @Override
     protected DocumentMigrationDescriptor createFromBaseObject(BaseObject object)
     {
-        ExtensionId extensionId = new ExtensionId(
```

Comment:
This seems to be formatting changes induced by your IDE, it's fine to commit them when you do actual modifications on the file itself, but it's not needed to commit a file just for its formatting changes.